quickwit-serve/src/lib.rs

```diff
@@ -112,7 +114,14 @@ pub async fn serve_quickwit(
         // Wait 10 seconds for nodes running a `Metastore` service.
         cluster
             .wait_for_members(has_node_with_metastore_service, Duration::from_secs(10))
-            .await?;
+            .await
+            .map_err(|_| {
+                error!(
```

Comment:
Nitpick: I think the helpline `Try unning quickwit with additional service `--service metastore` will be better if we move it in the `anyhow!` error. because that's the error intended for users .